### ISSUE TITLE
Make new shelf for the fridge

### ISSUE THREAD

**Issue by jackie1050** [DoESLiverpool/somebody-should#1049]
I found I had another piece of 10mm acrylic - we should make a new shelf to replace the remaining original shelf - although this is only slightly cracked at the moment, acrylic will be stronger and allow us to cool more cans for example...
(see #549 although there doesn't appear to be a related DXF)

**Comment by johnmckerrell** [comment]
![2019-05-22 12 50 51](https://user-images.githubusercontent.com/27243/62458523-130a2780-b775-11e9-9b03-0379569e4fa3.jpg)

A photo of the existing shelf which one might use to make a new one.

**Comment by johnmckerrell** [comment]
![fridge-shelf-jmc](https://user-images.githubusercontent.com/27243/62459970-95e0b180-b778-11e9-80f6-64d8a0bd43ef.png)

First pass at tracing the above photo, overall shelf dimensions are 320x510mm

**Comment by johnmckerrell** [comment]
![2019-08-05 12 49 46](https://user-images.githubusercontent.com/27243/62465174-312c5380-b786-11e9-9f36-47ffe7edc1da.jpg)

I found the shadows on the previous picture may have been confusing me so I tried drawing around the outside of the shelf and taking a photo of that. I've been trying and failing to simplify the image though and have now run out of time so if anyone else wants to try that would be appreciated. I didn't get the photo dead centre so it's slightly rotated and skewed, I'm sure that's something a clever person can fix in post-production!

**Comment by ajlennon** [comment]
I've designed a new shelf from @johnmckerrell 's drawing in OnShape.

The original design is here. Anybody using this link should be able to edit

https://cad.onshape.com/documents/422ee2d12c845233e3832762/w/b39e844a39dff7df64c1d1a9/e/6a1de4b16fad2186cb749737

NB If you edit and export this use the bottom rather than the top as otherwise you lose the curve.

The exported .DXF is in somewhere-safe here and can be importer into LaserCut as-is.

https://github.com/DoESLiverpool/somewhere-safe/blob/master/designs/kitchen/Fridge%20Shelf.dxf

I cut it out under Jackie's watchful eye and we've installed the new shelf into the 'fridge.

Apologies as no doubt everything in there is not where it once was!
